src/component/Button.svelte

```diff
@@ -0,0 +1,48 @@
+<script>
+    export let buttonName;
+    export let buttonSizeBig;
+    export let buttonSizeMedium;
+</script>
+
+<main>
+    <button
+        class="btn"
+        class:button-size-big={buttonSizeBig}
+        class:button-size-medium={buttonSizeMedium}
+        >{buttonName}
```

Comment:
bu tarz component'larda slot kullanmak daha iyi olur, gercek Button'a daha benzer bir deneyim sunar. Ilerde Button'in icinde textten baska atiyorum bi de icon gosterecek olursan daha rahat yapabilirsin.

Reply:
Buton ismini slot içine koydum. 
Class'ları değişkene bağlı olarak bıraktım. Çünkü slot'a bağlasaydım çağırdığım sayfalarda style'larını tekrar yazmam gerekiyordu. Aynı kodları tekrar tekrar yazmamak için bu şekilde bıraktım. Ama değiştirmemi istersen bunları da düzeltebilirim.